templates/index.html

```diff
@@ -119,6 +119,10 @@ <h1 id="logo"><b><i>Py</i>Format</b> Using <i>%</i> and <i>.format()</i> for gre
             </div>
             <footer>
                 <p>
+                    You might also like <a href="http://strftime.org/">Python strftime reference</a>
+                    by <a href="https://twitter.com/mccutchen">Will McCutchen</a>
```

Comment:
:+1: , thanks :) Could you just please add a full stop at the end of the sentence after the closing </a>? :)